_data/sites.json

```
@@ -12788,6 +12788,18 @@
         ]
     },
 
+    {
+        "name": "Minehut",
+        "url": "https://app.minehut.com/support/form",
+        "difficulty": "hard",
+        "notes": "The only way to contact them is at https://app.minehut.com/support/form and https://support.minehut.com/hc/en-us/requests/new.",
```

```suggestion
        "notes": "The only way to contact them is at the linked form or through [here](https://support.minehut.com/hc/en-us/requests/new).",
```

We try to reduce the number of links in the notes as much as possible.